.eslintrc.js

```diff
@@ -1,3 +1,14 @@
 module.exports = {
-  extends: 'algolia'
+  extends: 'algolia',
+  parserOptions: {
+    ecmaFeatures: {
+      legacyDecorators: true,
+    },
+  },
+  rules: {
+    'new-cap': [
```

Comment:
this file isn't actually used, but sometimes my eslint was enabled in TypeScript files anyway, and would error unless I specifically turned it off. We are still using tslint in this project